sanic_routing/route.py

```diff
@@ -200,17 +203,14 @@ def add_parameter(
             pattern = re.compile(pattern)
 
         is_regex = label not in self.router.regex_types
-        priority = (
-            0
-            if is_regex
-            else list(self.router.regex_types.keys()).index(label)
-        )
+        priority = 0 if is_regex else list(self.router.regex_types.keys()).index(label)
+
         self._params[idx] = param_info_class(
             name=name,
             raw_path=raw_path,
             label=label,
             cast=cast,
-            pattern=pattern,
+            pattern=pattern,  # type: ignore
```

Comment:
?

Reply:
``Argument "pattern" to "ParamInfo" has incompatible type "Union[Pattern[Any], str, None]"; expected "Pattern[Any]"``

Should I use cast to solve it?